ai_council/orchestration/layer.py

```
@@ -482,19 +485,25 @@ async def process_request(self, user_input: str, execution_mode: ExecutionMode,
             
             # Stage 6: Arbitration
             try:
-                validated_responses, explanation,arbitration_decisions = await self._stage_arbitrate(successful_responses)
+                validated_responses, explanation, arbitration_decisions = await self._stage_arbitrate(successful_responses)
 
             except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)